### DISCUSSION TITLE
Table with Entry has no OnSelected?

### DISCUSSION THREAD

**Discussion by Dmitriy147** [fyne-io/fyne#5150] [General]
How can i get such events  OnSelected and OnUnselected events with cells widget.Entry?
If i will replace in my table cell type from Entry to Label - all works fine!

**Comment by andydotxyz** [comment]
The Entry needs tap events to get the focus and show cursor. 
You can extend the Entry so that it selects the cell and then gains focus instead.

**Comment by Dmitriy147** [comment]
Thanks! Its work now.
